packages/astro/src/core/messages.ts

```diff
@@ -235,10 +235,10 @@ export function formatErrorMessage(err: ErrorWithMetadata, args: string[] = []):
 		args.push(red(padMultilineString(err.frame, 4)));
 	}
 	if (args.length === 1 && err.stack) {
-		args.push(dim(cleanErrorStack(err.stack)));
+		args.push(dim(err.stack));
 	} else if (err.stack) {
 		args.push(`  ${bold('Stacktrace:')}`);
-		args.push(dim(cleanErrorStack(err.stack)));
+		args.push(dim(err.stack));
```

Comment:
`cleanErrorStack` is removed as it should already be cleaned by `collectErrorMetadata`.